codebuild.go

```diff
@@ -1617,7 +1617,15 @@ func getUnderlying(pkg *Package, typ types.Type) types.Type {
 }
 
 func (p *CodeBuilder) findMember(
-	typ types.Type, name, aliasName string, flag MemberFlag, arg *Element, srcExpr ast.Node) MemberKind {
+	typ types.Type, name, aliasName string, flag MemberFlag, arg *Element, srcExpr ast.Node, visited map[types.Type]struct{}) MemberKind {
+	if visited == nil {
```

Comment:
move line `1621..1628` to func `embeddedField`

Reply:
done https://github.com/goplus/gogen/compare/900c0aa79477d8d2a8416e546bcd8d2dbf8b6412..b5ebdba3b5c81e378ade56129f68b0ab3ba0d950#diff-80795659b60b1d14b5a8d42dcfa8cade5f10e4846987ab0f3c6b22d012500d2c